### DISCUSSION TITLE
update cluster-operator from 0.20 to 0.22

### DISCUSSION THREAD

**Discussion by islet5** [strimzi/strimzi-kafka-operator#7560] [Q&A] [answered]
Hello,

I was trying to update jaeger from 1.20 to 1.37, but it seems I have to update kafka frist: 
![image](https://user-images.githubusercontent.com/9824473/199147102-06527d98-50dc-4849-878f-da40ddcb7956.png)

https://github.com/jaegertracing/jaeger-operator/issues/1371
![image](https://user-images.githubusercontent.com/9824473/199147020-cd113210-2901-4b50-a6db-1c8be7c5046c.png)

Then I try to update kafka cluster operator from 0.20 to 0.22, but my K8S is aks and version is already v1.22.6,
![image](https://user-images.githubusercontent.com/9824473/199147405-534e08d8-c95a-4e88-a5b8-6ef83cefb38c.png)

![image](https://user-images.githubusercontent.com/9824473/199147454-6559d227-ac1d-4a18-b752-08a6b17a72e9.png)

Can somebody tell me what should I do now 😭

**Comment by scholzj** [comment, marked as answer]
Strimzi 0.20 or 0.22 does not support Kubernetes 1.22. You should have upgraded it before upgrading Kubernetes. The only clean and tested way how to handle it would be to downgrade your Kubernetes, proceed with the Strimzi upgrade to 0.22 and 0.23 and then upgrade Kubernetes to 1.22. If that is not possible, you can probably try things such as do non-cascading delete of the CRs / CRDs and installing some new Strimzi version. But you should test it first to make sure how it works and avoid some data loss.
